### ISSUE TITLE
Window dropdown for plugins not keyboard navigable

### ISSUE THREAD

**Issue by mferrarini** [ProjectMirador/mirador#3903]
### Description
Initial testing indicates interface elements like Download PDF, Share, and Show Image Tools are not accessible through keyboard navigation, just by mouse. 

I was able to navigate through the "windows options" menu items with voiceover using ctrl+opt+left/right arrows, but this same combination does not work for me when voiceover is turned off. Did not work with NVDA either. This follows the same pattern of what we saw when investigating keyboard navigation through the info panels.

### Problem
The list items are given a tab-index of "-1" and therefore are being passed over when tabbing through the page.

### Possible solution
Removing the tabIndex="=1" from the `<ListSubheader>` material UI component should be all that's needed to make this work (and might also solve the problem in other areas as well). 

I noticed this component in the following files:
- [WindowThumbnailSettings.js](https://github.com/ProjectMirador/mirador/blob/master/src/components/WindowThumbnailSettings.js#L56)
- [WindowTopMenu.js](https://github.com/ProjectMirador/mirador/blob/master/src/components/WindowTopMenu.js#L14)
- [WindowViewSettings.js](https://github.com/ProjectMirador/mirador/blob/master/src/components/WindowViewSettings.js#L83)

**Comment by lutzhelm** [comment]
I'm not sure if #3914 fixes this entirely; I will have to try this with an actual plugin installed. But the window view settings / thumbnail navigation options are now keyboard accessible.

**Comment by marlo-longley** [comment]
Audit 2026: action -- check if this is still an issue in Mirador 4
Can use Mirador integration repo
